src/types.ts

```diff
@@ -275,6 +276,10 @@ export interface Finished {
   status: 'finished'
   context: Context
   value: Value
+  representation?: Representation // if the returned value needs a unique representation,
+  // (for example if the language used is not JS),
+  // the display of the result will use the representation
+  // field instead
```

Comment:
I'm thinking if we can even use this to handle #1124

Reply:
Sorry, could you elaborate? 😅 Representations currently only concern the final return value of a program - for now, for functions like display() or display_list(), the output remains unaffected